File: MCAuthClient/MCAuthClient/AuthAttempt.cs
﻿using System;

namespace Test
{
    public class AuthAttempt
    {
        public string ip { get; set; }
        public string auth { get; set; }
        public string uuid { get; set; }

        public AuthAttempt(string ip, string auth, string uuid)
        {
            this.ip = ip;
            this.auth = auth;
            this.uuid = uuid;
        }
    }
}

File: MCAuthAPI/src/main/java/me/leoata/MCAuthAPI/restservice/auth/attempt/AuthAttempt.java
package me.leoata.MCAuthAPI.restservice.auth.attempt;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
public class AuthAttempt {

    @Getter private String auth;
    @Getter private String ip;
    @Getter private String uuid;


}
